sdk/servicebus/Azure.Messaging.ServiceBus/tests/ReceiverLiveTests.cs

```diff
@@ -125,12 +125,12 @@ public async Task Receive_Event(int numThreads)
                 // use double the number of threads so we can make sure we test that we don't
                 // retrieve more messages than expected when there are more messages available
                 await sender.SendBatchAsync(GetMessages(numThreads * 2));
-                await using var receiver = new ServiceBusReceiverClient(
+                await using var processor = new ServiceBusProcessorClient(
```

Comment:
Is there a plan to move these to a `ProcessorLiveTests` class, or should we consider renaming this one?

Reply:
> I see all the tests are working except the ones where you pass `numThreads` as a parameter.
> 
> @JoshLove-msft Could you please check these 2 tests `Receive_Event(int numThreads)` and `Receive_Event_Consumes_All_Messages(int numThreads)`?

They are working locally. I will add a timeout to our tests to avoid hanging the entire run.

Reply:
> Is there a plan to move these to a `ProcessorLiveTests` class, or should we consider renaming this one?

Moved